releasenotes/notes/0.8.0/add-universal-functions-1e54351f1f7afa4b.yaml

```
@@ -1,7 +1,7 @@
 ---
 features:
   - |
-    New :ref:`universal-functions` that can take in a :class:`~retworkx.PyGraph`
+    New `universal-functions` that can take in a :class:`~retworkx.PyGraph`
```

This will render kind of weirdly because it would basically be:

New _universal-functions_ that

so let's just drop the reference formatting and go with:

```suggestion
    New universal functions that can take in a :class:`~retworkx.PyGraph`
```